cpp/arcticdb/version/version_store_api.hpp

```diff
@@ -170,7 +171,8 @@ class PythonVersionStore : public LocalVersionedEngine {
             bool append,
             bool convert_int_to_float,
             bool via_iteration,
-            bool sparsify
+            bool sparsify,
+            bool prune_previous_versions = false
```

Comment:
Why is this the only one with a default???

Reply:
Yes misread that. It is unnecessary